docs/deployment/index.rst

```diff
@@ -0,0 +1,266 @@
+   popd
+
+To build the ``rest_api_flint.example`` image locally, run the following
+command:
+
+.. code:: sh
+
+   pushd rest_api_flint.example
+   docker build -t flint-api .
+   popd
+
+Create a resource group
+~~~~~~~~~~~~~~~~~~~~~~~
+
+You can now prepare resources in Azure to which you can push the image
+and then deploy a container to Azure App Service. You can start by
+creating a resource group in which you want to collect all the
+resources.
+
+.. code:: sh
+
+   az group create --name myResourceGroup --location centralindia
+
+You can change the ``--location`` value to specify a region near you.
+
+Create a Container Registry
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can now push the image to Azure Container Registry from which App
+Service can deploy it. Create an Azure Container Registry to push your
+images to:
+
+.. code:: sh
+
+   az acr create --name <registry-name> --resource-group myResourceGroup --sku Basic --admin-enabled true
+
+Replace ``<registry-name>`` with a suitable name for your registry. The
+name must contain only letters, numbers, and must be unique across all
+of Azure.
+
+Retrieve your credentials for the Container Registry:
+
+.. code:: sh
+
+   az acr credential show --resource-group myResourceGroup --name <registry-name>
+
+Use the ``docker login`` command to sign in to the container registry:
+
+.. code:: sh
+
+   docker login <registry-name>.azurecr.io --username <registry-username>
+
+Replace ``<registry-name>`` and ``<registry-username>`` with values from
+the previous steps. When prompted, type in one of the passwords from the
+previous step.
+
+Let's tag the images with the registry name:
+
+.. code:: sh
+
+   docker tag rest_api_gcbm <registry-name>.azurecr.io/rest_api_gcbm:latest
+   docker tag rest_api_flint.example <registry-name>.azurecr.io/rest_api_flint.example:latest
+
+Use the docker push command to push the image to the registry:
+
+.. code:: sh
+
+   docker push <registry-name>.azurecr.io/rest_api_gcbm:latest
+   docker push <registry-name>.azurecr.io/rest_api_flint.example:latest
+
+Use the ``az acr repository list`` command to verify that the push was
+successful:
+
+.. code:: sh
+
+   az acr repository list -n <registry-name>
+
+Deploy the image from registry
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To deploy a container to Azure App Service, you first create a web app
+on App Service, then connect the web app to the container registry. When
+the web app starts, App Service automatically pulls the image from the
+registry.
+
+Create an App Service plan using the ``az appservice plan create``
+command:
+
+.. code:: sh
+
+   az appservice plan create --name myAppServicePlan --resource-group myResourceGroup --is-linux
+
+Create the web app with the ``az webpp create`` command. Since we are
+deploying two images to two different web apps, you would need to
+specify the commands twice. To deploy, ``rest_api_gcbm`` to the first
```

Comment:
```suggestion
Create the web app with the ``az webpp create`` command. Since we are
deploying two images to two different web apps, you need to
enter these commands twice. To deploy ``rest_api_gcbm`` to the first
```